enterprise/backend/src/metabase_enterprise/serialization/dump.clj

```diff
@@ -24,11 +25,41 @@
 
 (set! *warn-on-reflection* true)
 
+(def ^:private serialization-order
+  (delay (-> (edn/read-string (slurp (io/resource "serialization-order.edn")))
+             (update-keys name)
+             (update-vals (fn [order]
+                            (into {} (map vector order (range))))))))
+
+(def ^:private serialization-sorted-map
+  (memoize
+   (fn [t]
```

Comment:
Let's give the parameters meaningful names like `model-name`. I thought this was memoizing the maps being stored for a second, then realized this is memoizing the "model -> empty map with the right sort order" function.

Reply:
Done